### ISSUE TITLE
Request: Add to negative prompts from normal prompt wildcards

### ISSUE THREAD

**Issue by qrrbszopgu** [adieyal/sd-dynamic-prompts#89]
Would be nice to have some syntax to indicate things that should be added to the negative prompt if a certain wildcard is chosen. 

Ex: For a clothing wildcard file that had "denim shorts" on a line, I would indicate on that line that "denim" should also be added to the negative prompt so I don't get results of people dressed head to toe in denim.

**Comment by adieyal** [comment]
would you then simply add denim to the end of the existing negative prompt?

**Comment by qrrbszopgu** [comment]
Yes ideally that's what it would do. I always use a general purpose negative prompt, but some positive prompts don't work well without specific negative prompts added to that.

**Comment by adieyal** [comment]
I'm going to keep this ticket open for a while to see if others need this as well. Given that it requires new syntax it may result in addition complexity and I'm hesitant to implement it if it is a marginal use case

**Comment by JohnWJarrett** [comment]
I'd like this feature too

**Comment by platypusq** [comment]
I would also appreciate this feature

**Comment by adieyal** [comment]
on my todo list :)

**Comment by penngwynXVII** [comment]
I agree that this would be a good feature.

**Comment by adieyal** [comment]
Any suggestions for how to implement this? Would you have something like a `||` separating positive and negative ,e.g.

clothing.txt
```
...
denim skirt||denim
...
```

also, is there consensus that the negative keyword should be appended to the end of the negative prompt?

**Comment by JohnWJarrett** [comment]
Yeah, the || seems like it would be great, I don't use a blank space anywhere except the end "{cow|cat|}" for example, so I don't see any reason it should mess with anything else, though would it still be useable inline, so with my previous example "{cow|cat||lion|}"
I do usually use wildcard files, but sometimes for testing, an inline is great...

as for positioning, I think that if the position mattered, it would probably need to be in the main negative in the first place, but that's just me, though you could have it be "denim skirt||#denim" to be at the start and "denim skirt||denim#" for the end could work, or something along those lines, and have nothing to just have it fall wherever it falls... or some other symbol, I just used hash for clarity sake here, maybe an exclamation mark or an astrix

**Comment by acorderob** [comment]
I think using an exclamation mark as the separator for the negative value would be better since i don't think it is used for any other feature. It also usually means "not" in programming languages. You could also allow a "<" or ">" character after it to indicate where to add the text, the start or end of the negative prompt.

**Comment by acorderob** [comment]
Bump. I am, more and more, wishing for this feature in multiple situations.

**Comment by acorderob** [comment]
For anyone interested, I made my own extension to manage this: https://github.com/acorderob/sd-webui-sendtonegative

**Comment by qrrbszopgu** [comment]
> For anyone interested, I made my own extension to manage this: https://github.com/acorderob/sd-webui-sendtonegative

that's perfect, thanks for making it
